### PR TITLE
C3js support

### PR DESCRIPTION
**Pull request by ppikula**
It still needs some love regarding the colors, styles, axis description. Thanks to this lib we got rid of quite big portion of code.

Please try to use it and tell me if it is OK. The biggest advantage of this lib is it is maintained and  has react support.

### CODE REVIEW

File: priv/app/graph.jsx [hunk 1]
```diff
@@ -47,6 +41,35 @@ export default class Graph extends React.Component {
...
 
+    const data = {
+      x: "x",
+      columns: this.state.columns,
+      axes: {
+        count: "y2"
+      },
+      colors: {
+        count: "#98fb98",
+        max: "#8c2a04",
```
Comment (afronski) [inline priv/app/graph.jsx]: Let's use `"` everywhere here then.
And please use uppercase letters for colors. 😉

File: priv/app/graph.jsx [hunk 2]
```diff
@@ -151,8 +168,29 @@ export default class Graph extends React.Component {
...
 
-  chartId() {
-    return Utils.chartId(this.props.mfa);
+  createColumns(data) {
+    let columns = [
+      [ "x" ], [ "count" ], [ "max" ], [ "99th perc" ], [ "90th perc" ],
+      [ "75th perc" ], [ "50th perc" ], [ "mean" ], [ "min" ]
+    ];
+    let zu = function(v) {
+      if (v === undefined) {
```
Comment (afronski) [inline priv/app/graph.jsx]: Use `typeof` and lambda syntax.

File: priv/app/graph.jsx [hunk 3]
```diff
@@ -151,8 +168,29 @@ export default class Graph extends React.Component {
...
+        return 0;
+      } else {
+        return v;
+      }
+    };
+    for (let d of data) {
```
Comment (afronski) [inline priv/app/graph.jsx]: Maybe `entry` instead of `d`?

File: priv/package.json [hunk 4]
```diff
@@ -18,12 +18,13 @@
   },
   "homepage": "https://github.com/appliscale/xprof",
   "dependencies": {
-    "bootswatch": "3.3.7",
     "bootstrap": "3.3.7",
+    "bootswatch": "3.3.7",
     "jquery": "~2.1.4",
```
Comment (afronski) [inline priv/package.json]: Let's remove `~`.

**Comment by ppikula** [comment]
closes #38

**Pull request merged**